Demo/Common/Networking/PaymentsRequest.swift

```diff
@@ -31,14 +31,17 @@ internal struct PaymentsRequest: Request {
         
         try container.encode(data.paymentMethod.encodable, forKey: .details)
         try container.encode(data.storePaymentMethod, forKey: .storePaymentMethod)
+        try container.encode(data.shopperName, forKey: .shopperName)
+        try container.encode(data.emailAddress ?? Configuration.shopperEmail, forKey: .shopperEmail)
+        try container.encode(data.telephoneNumber, forKey: .telephoneNumber)
+        try container.encode(Locale.current.identifier, forKey: .shopperLocale)
```

Comment:
Is this safe? What if shopper have dutch locale but wants english?
just wandering..

Reply:
well, merchants can have a settings page for the shopper to choose the locale, its totally up to merchants, right?